conference_101719.tex

```diff
@@ -0,0 +1,200 @@
+
+\subsubsection{Preparation for the detection}
+MAC addresses and SSIDs of APs are collected through beacon frames which exist in the communication range of a user device in preparation for the detection.
+Among them, APs whose SSIDs are identical to that of the targeted AP, namely \newcommand{\SSID}{S_{A_{target}}}$\SSID$, are extracted and represented as $AP_{all}=\{A_{input}|0\le input \le n_{all}\}$, where $n_{all}$ .
+
+\subsubsection{Flowchart of Our System}
+The algorithm consists of four procedures as shown in \ref{fig:flowchart}, which are 1) Comparison with $\tarMAC$, 2)Setting $\userMAC$ to \newcommand{\inputMAC}{M_{A_{input}}}$\inputMAC$ , 3)ARP request from a user device to a gateway and 4)Observation of ARP reply from the gateway to the user device.
+Note that they are repeatedly conducted for every MAC address in $APP_{all}$ unless a RAP is detected.
+
+\subsubsection{Comparison with $\tarMAC$}
+In the first phase, our system compares the $\inputMAC$ with the $\tarMAC$.
+If they are the same MAC address, namely $\inputMAC = \tarMAC$, it easily judges the either of them is RAP due to the cloned address.
+Thus, using the network should be avoided whether it is on the same path or not because the risk of being attacked is extremely high. 
+However, in the case where a RAP clones the MAC address which the LAP far from the user device has or it does not clone, our system cannot detect only by checking the accordance of MAC addresses.
+Thus, the detection process goes to the next phase.
+
+\subsubsection{Setting $\userMAC$ to $\inputMAC$}
+In this phase, in order to reveal the existence of LAP which is connected on the other side of the targeted AP if the user device is attacked, $\userMAC$ is set to $\inputMAC$ in preparation for the next procedure.
+
+\subsubsection{ARP Request from the User Device to the Gateway}
+After the MAC address is set to $\inputMAC$, the connection between the user device and $\tarAP$ is once lost since the gateway become unable to use original MAC address as a destination for the packets.
+Thus, the user device needs to send ARP request in order to reconnect with it through the $\tarAP$ automatically due to the stronger RSSI than any other $AP_{all}$.
+
+\subsubsection{Observation of ARP Reply from the Gateway to the User Device}
+Finally, in this procedure, the ARP reply packets, whose destination is set as $\inputMAC$, are observed in order to investigate whether the $\inputAP$ is on the same path with the user device.
+If the packets do not reach the user device, the $\tarAP$ can be revealed as a RAP, which exists between the user device and the $\inputAP$ and the detection procedure is finished.
+Otherwise, since the $\tarAP$ is judged that it is on the distinct path from the $\inputAP$ , the flow are repeated from the beginning with the next $\inputAP$ until the all $\inputMAC$s of $AP_{all}$ are input.
```

Comment:
```suggestion
Otherwise, the $\tarAP$ is judged that it is on the distinct path from the $\inputAP$.
In this case, the same procedures in the flowchart are repeatedly conducted for another $\inputAP$ in $\M_{all}$ until the RAP is detected.
```